test/hexagon/apps/dilate3x3/dilate3x3.cpp

```diff
@@ -2,7 +2,8 @@
 #include "halide-hexagon-setup.h"
 #include <stdio.h>
 using namespace Halide;
-
```

Comment:
Oh wait, I thought this was test/correctness/dilate3x3.cpp. It's fine to leave it :)